### DISCUSSION TITLE
Parameter VOLTAGE_DROP doesn't work for me

### DISCUSSION THREAD

**Discussion by johnnybee** [Louisvdw/dbus-serialbattery#742] [Q&A]
Hello,
I have a daly bms which is connected to a multiplus II. I'm using the Mulitplus as battery monitor. Everything works as expected, but there is an voltage difference of about 2V between the daly bms (lower voltage) and the multiplus (higher voltage). 
The voltage the daly bms measure is the right one, which I have checked with a measuring device voltmeter.
Now I thought the parameter VOLTAGE_DROP is exactly for this case. So I used "VOLTAGE_DROP = 2.00" in my config.ini.
But it doesn't make any difference, if I use the parameter or not. 
If I look in the parameters of the daly battery in my venus OS, it's always the same charge voltage limit no matter if I use the parameter or not.
The problem now is when the multiplus reachs 56V, the daly bms slow down charging, but itself says something like 54V and 65% fuel. 
What am I doing wrong? Have I missed another corressponding parameter?
Before I forget the important informations: I use Venos OS V3.00 on a raspberry pi 3b and the latest serialbattery drive 1.0.20230531.

Thank you for your help!

**Comment by mr-manuel** [comment]
If you have a voltage difference of 2 V I would say that there is something wrong with your setup.
Please install the nightly version of the dev branch.

**Reply by johnnybee** [reply]
Hello Manuel,

I'll give it a try and will tell. Thank you for the idea!
